### ISSUE TITLE
minknow_api cannot be imported

### ISSUE THREAD

**Issue by mattloose** [nanoporetech/minknow_api#36]
The latest release of minknow_api (version 5.0.0) cannot be imported.

Steps to reproduce:

In a clean python3 environment run:

```bash
pip install minknow_api
python -c "import minknow_api"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/matt/testenv/lib/python3.7/site-packages/minknow_api/__init__.py", line 87, in <module>
    from minknow_api.manager import get_local_authentication_token_file
  File "/Users/matt/testenv/lib/python3.7/site-packages/minknow_api/manager.py", line 21, in <module>
    import minknow_api.basecaller_service
ModuleNotFoundError: No module named 'minknow_api.basecaller_service'
```

**Comment by jorj1988** [comment]
Thanks @mattloose  we will investigate internally and have a look, I suspect something has gone wrong with our deploy to pypi.

In the mean time, building the package from github should work as normal.

- George

**Comment by mattloose** [comment]
Hi - thanks george - we can confirm that building from github works.

Matt

**Comment by amerry** [comment]
Apologies for that. I've uploaded a 5.0.0.1 package, which should be fixed (the code is the same as the 5.0.0 tag in the repository).

**Comment by mattloose** [comment]
Thanks.
